kong/plugins/apifortress/schema.lua

```diff
@@ -0,0 +1,23 @@
+-- The plugin schema
+
+local function validateKeys(value)
+  return #value > 0
+end
+
+local function validateThreshold(value)
+  return value > 0
+end
+
+local function validateProjectId(value)
+  return value > 0
+end
+
+return {
+  fields = {
+    apikey = { required = true, type = "string", func = validateKeys },
+    secret = { required = true, type = "string", func = validateKeys },
+    endpoint = {required = true, type = "string", func = validateKeys },
```

Comment:
According to the documentation, this property has a default for the cloud version of your platform. It could be helpful to provide this default in the plugin's schema, so that users only need to specify this property when using an on-prem solution (like the other logging/monitoring plugins of Kong).